docs/resource_types/additional_nics.md

```diff
@@ -17,8 +17,8 @@ resource "ddcloud_additional_nics" "additional_nic_test" {
 The following arguments are supported:
 
 * `server` - (Required) ID of the server to which the network adapter needs to be added
-* `private_ipv4` - (Optional) IPv4 Address for the new network adapter. Exactly one of `private_ipv4` or `vlan_id` or both must be specified.
-* `vlan_id` - (Optional) VLAN ID of the new network adapter. Exactly one of `private_ipv4` or `vlan_id` or both must be specified.
+* `private_ipv4` - (Optional) IPv4 Address for the new network adapter. Exactly one of `private_ipv4` or `vlan` or both must be specified.
+* `vlan` - (Optional) VLAN ID of the new network adapter. Exactly one of `private_ipv4` or `vlan` or both must be specified.
```

Comment:
Just want to check on the wording here - could this be simplified to say, "you must specify _at least one_ of `private_ipv4` or `vlan`"? At the moment I'm not sure if it's saying that you can only specify one or the other (but not both).